src/server/rest/v1/validator/AuthValidation.ts

```diff
@@ -63,7 +63,7 @@ export default class AuthValidator extends SchemaValidator {
     return data;
   }
 
-  public validateAuthResendVerificationEmail(data: any): Partial<HttpResendVerificationMailRequest> {
+  public validateAuthResendVerificationEmail(data: HttpResendVerificationMailRequest): Partial<HttpResendVerificationMailRequest> {
```

Comment:
Please correct me if I'm wrong, but I think we can't type `data` here. As it is a user input, we can't determine which type it is before processing the validation. Same for typing below. What do you think ?

Reply:
> Please correct me if I'm wrong, but I think we can't type `data` here. As it is a user input, we can't determine which type it is before processing the validation. Same for typing below. What do you think ?

Normally you should expect to receive the type HttpResendVerificationMailRequest, nothing else so you can type it.
It's better than 'any'.
What's the problem of typing it here?

Reply:
Ok, I thought that as anything could be sent through the endpoint, we were not supposed to anticipate the structure of the payload (as the any "request" param in each security class) until the validation is processed.

Reply:
We basically validate an already structured request in that code but when the input is a `QueryString.ParsedQs` express type, it's difficult to make the input type match the output type. I think some `Http...Request` type definition need some refinements to allow them to be reused transparently with express `Request` attributes.